mirdata/datasets/phenicx_anechoic.py

```diff
@@ -0,0 +1,422 @@
+
+        """
+        return load_score(self.notes_path)
+
+    @core.cached_property
+    def notes_original(self) -> Optional[annotations.NoteData]:
+        """the track's notes corresponding to the original score
+
+        Returns:
+            NoteData: Note data for the track
+
+        """
+        return load_score(self.notes_original_path)
+
+    def get_audio_voice(self, id_voice: int) -> Optional[Tuple[np.ndarray, float]]:
+        """the track's audio
+
+        Returns:
+            * np.ndarray - the mono audio signal
+            * float - The sample rate of the audio file
+
+        """
+        assert id_voice < self.n_voices
+        return load_audio(self.audio_paths[id_voice])
+
+    def to_jams(self):
+        """Get the track's data in jams format
+
+        Returns:
+            jams.JAMS: the track's data in jams format
+
+        """
+        return jams_utils.jams_converter(
+            audio_path=self.audio_paths[0], note_data=[(self.notes, "aligned notes")]
+        )
+
+
+class MultiTrack(core.MultiTrack):
+    """Phenicx-Anechoic MultiTrack class
+
+    Args:
+        mtrack_id (str): track id of the track
+        data_home (str): Local path where the dataset is stored.
+            If `None`, looks for the data in the default directory, `~/mir_datasets/Phenicx-Anechoic`
+
+
+    Attributes:
+        audio_path (str): path to the audio files associated with this track
+        annotation_path (str): path to the annotations files associated with this track
+        mtrack_id (str): multitrack id
+        tracks (list): list of track ids within this multitrack
+    """
+
+    def __init__(
+        self,
+        mtrack_id,
+        data_home,
+        dataset_name,
+        index,
+        track_class=Track,
+        metadata=None,
+    ):
+        super().__init__(
+            mtrack_id,
+            data_home,
+            dataset_name,
+            index,
+            Track,
+            metadata,
+        )
+
+        #### parse the keys for the dictionary of instruments and strings
+        self.instruments = {
+            source.replace(self.mtrack_id + "-", ""): source
+            for source in self.track_ids
+        }
+        self.sections = {"brass": [], "strings": [], "woodwinds": []}
+        for instrument, track_id in self.instruments.items():
+            self.sections[DATASET_SECTIONS[instrument]].append(track_id)
+
+        self.piece = self.mtrack_id
+
+    @property
+    def track_audio_property(self):
+        #### the attribute of Track which returns the relevant audio file for mixing
+        return "audio"
+
+    def get_audio_for_instrument(self, instrument):
+        """Get the audio for a particular instrument
+
+        Args:
+            instrument (str): the instrument to get audio for
+
+        Returns:
+            np.ndarray: instrument audio with shape (n_samples, n_channels)
+
+        """
+        assert instrument in self.instruments.keys()
```

Comment:
nit: I prefer raising errors over asserts, so the user gets a more clear message if something goes wrong, and its easier to test for.